snowflake_blueprints/upload_file.py

```diff
@@ -330,6 +373,12 @@ def main():
         folder_name=source_folder_name, file_name=source_file_name)
     table_name = args.table_name.upper()
     insert_method = args.insert_method
+    data_types = args.snowflake_data_types
+    if args.snowflake_data_types:
+        data_types = eval(args.snowflake_data_types)
```

Comment:
I'm not sure here, but in other places where I've had to use eval, `ast.literal_eval` was the safer bet. `eval` can result in code injection. Make sure?